frontend/src/store/modules/auth/actions.js

```diff
@@ -1,27 +1,25 @@
 import {LOGIN, LOGOUT} from './types/actions';
 import {SET_AUTHENTICATED_USER, USER_LOGIN, USER_LOGOUT} from "./types/mutations";
+import { authorize, me } from '../../../api/auth';
 
 export default {
     [LOGIN]: (context, user) => {
-        return new Promise((resolve, reject) => {
-            const fakeUser = {
-                email: 'test@gmail.com',
-                name: 'test user',
-                password: 'secretpassword',
-                access_token: 'jwt-auth-token',
-                id: 1
-            };
-
-            if(fakeUser.email === user.email && fakeUser.password === user.password) {
-                context.commit(USER_LOGIN, fakeUser);
-                context.commit(SET_AUTHENTICATED_USER, fakeUser);
-                resolve(fakeUser);
-            }
+        return authorize(user)
+          .then(response => {
+            me()
+              .then(response => {
+                const user = response[0];
+                context.commit(SET_AUTHENTICATED_USER, user);
+              })
+              .catch(error => {
```

Comment:
Not sure that you need to catch and throw same error, this catch is redundant, same for other catch